### DISCUSSION TITLE
Proposal for new backend endpoint

### DISCUSSION THREAD

**Discussion by skyzh** [skyzh/make-a-fortune#77] [Ideas]
「无可奉告」社区的所有用户，
你们好！很抱歉以这样一种突兀的方式和大家告别。
2021年3月22日，我们收到了一位用户的邮件，该用户向我们反映他已于3月16日报警，原因是该用户数次发现有人在一些“性相关”话题下留下他的QQ号，导致不断有人加该用户QQ进行骚扰，对该用户的个人名誉损害极大。该用户在邮件中向「无可奉告」团队询问警方是否有联系团队查找该用户。如果没有，该用户请求「无可奉告」团队协助查找，并由该用户上报派出所处理。
「无可奉告」团队后续第一时间对相应的违规言论和发言账号进行了屏蔽和禁言处理，并向该用户表示我们在技术上做不到查询用户真实身份。
2021年3月29日，「无可奉告」团队收到了来自闵行区派出所的电话。次日，「无可奉告」团队负责人于上午前往上海市公安局文保分局闵行高校派出所配合调查。警方根据《中华人民共和国网络安全法》对「无可奉告」进行了合规检查，要求「无可奉告」团队依法整改：1）后台信息实名制注册，2）完善论坛审查监督机制。
在与警方的交流之后，「无可奉告」团队认识到现阶段「无可奉告」的运营和管理形式存在相当多不合法不合规的地方。由于团队成员都是在校学生，精力有限，难以在科研和学业任务之外，继续承担一个社区论坛的运营，经过团队内部的讨论与交流，我们决定给「无可奉告」画上句号。
「无可奉告」将于4月11日21时关停，届时将清空所有相关数据。
感谢大家在「无可奉告」上线以来给予的支持和喜爱，再见！
「无可奉告」记录短片: www.bilibili.com/video/BV1Rp4y187ZJ

What about proposing a new backend for make-a-fortune?

**Comment by skyzh** [comment]
You may also view the message at https://make-a-fortune.vercel.app/posts/026581

**Comment by skyzh** [comment]
制度上可以考虑邀请制连带封号？

实现上可能的方法有：
1. 分布式 bot 接入已有的论坛后端
2. 提供 closed.social 的 RPC 接口

**Reply by hebingchang** [reply]
其实我觉得只要保证 后台实名 + 高强度审核 就没问题，现在比较流行的匿名论坛应该都是这个模式

但考虑到 高强度审核 是相当困难的，所以。。

**Comment by hebingchang** [comment]
Though it might be accused as fish in troubled waters, I think It's ok to develop a more modern backend.
